user/cron-jobs.md

```diff
@@ -9,11 +9,9 @@ permalink: /user/cron-jobs/
 
 <div id="toc"></div>
 
-Cron jobs run builds at regular scheduled intervals independently of whether
-any commits were pushed to the repository. They can run `daily`, `weekly` or `monthly`, which in practice means up to an hour after the selected time span. They cannot be set to run at specific times.
+Cron jobs run builds at regular scheduled intervals independently of whether any commits were pushed to the repository. When they run, cron jobs always fetch the most recent commit to a particular branch and build the project at that state. Cron jobs can run `daily`, `weekly` or `monthly`, which in practice means up to an hour after the selected time span, and they can also be [skipped](#Skipping-Cron-Jobs). Cron jobs cannot be set to run at specific times. 
```

Comment:
I'm not entirely sure whether we should be talking about "commits to a branch" or "commits of a branch" in this context. Thoughts?